common/sgx/sgxcertextensions.c

```diff
@@ -503,20 +509,49 @@ oe_result_t ParseSGXExtensions(
     {
         // There are two possible optional extensions. They are expected to be
```

Comment:
There are more than 'two' now.

Reply:
Yes, actually there are 4 or 5 optional extensions.
1st is platform instance id
2nd is configuration.
But rest 3 are sub extensions nested in configuration. So I'm not sure how to define the count. But I updated the comment to reflect this.